test/export/test_export.py

```diff
@@ -4269,8 +4296,6 @@ def forward(self, xs, y):
             )
 
     @testing.expectedFailureSerDer  # We don't preserve metadata on graph module
-    @testing.expectedFailureNonStrict
-    @testing.expectedFailureTrainingIRToRunDecompNonStrict
     def test_retrace_graph_level_meta_preservation(self):
```

Comment:
It seems like now we just consistently store static constraints for unspecified dims, like `{"t_id": ..., "dim": ..., "min": 4, "max": 4} so this doesn't break anymore. Any thoughts on this?